src/python/WMCore/MicroService/MSOutput/MSOutput.py

```diff
@@ -1023,3 +1027,20 @@ def alertDatatierNotFound(self, datatierName, containerName, isRelVal):
         self.logger.critical(alertDescription)
         if self.msConfig["sendNotification"] and not isRelVal:
             self.sendAlert(alertName, alertSeverity, alertSummary, alertDescription, self.alertServiceName)
+
+    def alertGenericError(self, caller, workflowname, msg, ex_str, document_str):
```

Comment:
Dario, please use lower "camelCase" variable names, as defined in the pylintrc:
https://github.com/dmwm/WMCore/blob/master/standards/.pylintrc#L104

Reply:
oops, you are correct, thanks!